_adr/83/index.adoc

```diff
@@ -0,0 +1,132 @@
+---
+num: 0 # allocate an id when the draft is created
+title: Permission API
+status: "Draft"
+authors:
+  - "Massimiliano Ziccardi"
+tags:
+  - "kafka"
+#applies_padrs: # What PADRs does this ADR apply?
+#applies_patterns: # What APs does this ADR apply?
+---
+## Context and Problem Statement
+Deciding whether a certain operation is allowed can be a complex matter and may require interactions with different components.
+
+Consumers of the Kas Fleet Manager APIs have no way of verifying in advance whether a particular request will be successful.
+The only way to do that, is to perform the same checks that Kas Fleet Manager will do while executing the action.
+
+This brings complexities and duplications. For example, just to check whether the creation of a new kafka is possible or not, at least the following checks must be performed:
```

Comment:
It's good to be explicit about where those duplications are. IIUC it's in the fleet manager and the API clients (UI and CLI) right?

Reply:
Yes. UI and CLI. Added into the ADR